src/bin/pg_autoctl/ipaddr.c

```diff
@@ -25,16 +26,22 @@
 
 #include "postgres_fe.h"
 
+#include "defaults.h"
 #include "env_utils.h"
 #include "file_utils.h"
 #include "ipaddr.h"
 #include "log.h"
+#include "string_utils.h"
 
 static unsigned int countSetBits(unsigned int n);
 static unsigned int countSetBitsv6(unsigned char *addr);
 static bool ipv4eq(struct sockaddr_in *a, struct sockaddr_in *b);
 static bool ipv6eq(struct sockaddr_in6 *a, struct sockaddr_in6 *b);
 static bool fetchIPAddressFromInterfaceList(char *localIpAddress, int size);
+static bool ipaddr_sockaddr_to_string(struct addrinfo *ai,
+									  char *ipaddr, size_t size);
+static bool ipaddr_getsockname(int sock, char *ipaddr, size_t size);
+
 
 /*
  * Connect in UDP to a known DNS server on the external network, and grab our
```

Comment:
This comment is outdated now. We don't connect to a DNS server on a postgres node and we don't use UDP either.

Reply:
Good point, fixed now!